plugin/pkg/scheduler/algorithm/priorities/selector_spreading.go

```diff
@@ -60,26 +60,26 @@ func NewSelectorSpreadPriority(
 }
 
 // Returns selectors of services, RCs and RSs matching the given pod.
-func getSelectors(pod *v1.Pod, sl algorithm.ServiceLister, cl algorithm.ControllerLister, rsl algorithm.ReplicaSetLister, ssl algorithm.StatefulSetLister) []labels.Selector {
-	selectors := make([]labels.Selector, 0, 3)
-	if services, err := sl.GetPodServices(pod); err == nil {
+func (s *SelectorSpread) getSelectors(pod *v1.Pod) []labels.Selector {
```

Comment:
The other thing is, extensibility and readability, everytime when someone makes changes, the arguments to this function increase which is weird as we can directly get from struct.